test/IDE/complete_decl_attribute.swift

```diff
@@ -307,6 +311,7 @@ func dummy2() {}
 // KEYWORD_LAST-DAG: Keyword/None:                       transpose[#Declaration Attribute#]; name=transpose
 // KEYWORD_LAST-DAG: Keyword/None:                       noDerivative[#Declaration Attribute#]; name=noDerivative
 // KEYWORD_LAST-DAG: Keyword/None:                       Sendable[#Declaration Attribute#]; name=Sendable
+// KEYWORD_LAST-DAG: Keyword/None:                       _unavailableFromAsync[#Declaration Attribute#]; name=_unavailableFromAsync
```

Comment:
These changes won't be needed if you do the `UserInaccessible` thing.

Reply:
Done